modules/apps/29-fee/types/fee.go

```diff
@@ -60,8 +60,10 @@ func NewFee(recvFee, ackFee, timeoutFee sdk.Coins) Fee {
 }
 
 // Total returns the total amount for a given Fee
+// The total amount is the Max(TimeoutFee, AckFee + RecvFee)
```

Comment:
maybe just a bit more explicit that only the max of these two will actually be escrowed?

Reply:
Maybe also indicate the reasoning for this decision (to efficiently escrow capital)?